default.nix

```diff
@@ -36,64 +40,50 @@ stdenv.mkDerivation rec {
   name = "coq";
 
   buildInputs = [
-
-    # Coq dependencies
     hostname
-  ] ++ (with ocamlPackages; [
-    ocaml
-    findlib
-    camlp5_strict
-    num
-
-  ]) ++ (if buildIde then [
-
-    # CoqIDE dependencies
-    ocamlPackages.lablgtk
-
-  ] else []) ++ (if buildDoc then [
-
+    python2 time # coq-makefile timing tools
```

Comment:
Why do we need `python2`? Can we contemplate porting that to python 3?

(I’m looking at the `python3` a few lines below…)